lib/uploadcare-rails.rb

```diff
@@ -0,0 +1,21 @@
+require 'uploadcare/configuration'
+
+module Uploadcare
+  module Rails
```

Comment:
Can we use this configuration style for other frameworks?
Is this Rails-specific?

Reply:
I have not worked with other frameworks, so hard to say if we can use this configuration style for them. At least I guess it framework specific and this is a reason not to place it to the ruby-gem